AUTHORS

```diff
@@ -0,0 +1,3 @@
+# This file lists all individuals having contributed content to the repository.
```

Comment:
is this really needed? contributor graph is there

Reply:
generated by Makefile

Reply:
ok, then

maybe another PR can remove it.

Reply:
I removed it